rubyxls.gemspec

```diff
@@ -27,5 +27,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
 
-  spec.add_runtime_dependency "axlsx", "~> 2.0.1"
+  spec.add_runtime_dependency "axlsx", "~> 2.0"
```

Comment:
How does this fix the vulnerability?  This just will let bundler resolve axlsx to >= 2.0 and < 3.0